academic-services/knowledge-exploration-service/index.yml

```diff
@@ -1,39 +1,65 @@
-### YamlMime:YamlDocument
-documentType: LandingData
-title: Microsoft Academic Knowledge Exploration Service (MAKES) documentation
+### YamlMime:Landing
+
+title: MAKES documentation
+summary: Microsoft Academic Knowledge Exploration Service (MAKES) offers a fast and effective way to add interactive academic search and refinement to applications. Learn about Microsoft Academic Knowledge Exploration Service, including technology overviews, API references, how-to guides, tutorials and more.
+
 metadata:
-  title: MAKES documentation
-  description: Learn about Microsoft Academic Knowledge Exporation Service, including technology overviews, API references, how-to guides, tutorials and more.
-  meta.description: Learn about Microsoft Academic Knowledge Exporation Service, including technology overviews, API references, how-to guides, tutorials and more.
+  title: Microsoft Academic Knowledge Exploration Service
+  description: Learn about Microsoft Academic Knowledge Exploration Service, including technology overviews, API references, how-to guides, tutorials and more.
   ms.topic: landing-page
-  ms.date: 2020-02-10
-abstract:
-  description: Microsoft Academic Knowledge Exploration Service offers a fast and effective way to add interactive academic search and refinement to applications. Learn about Microsoft Academic Knowledge Exporation Service, including technology overviews, API references, how-to guides, tutorials and more.
-sections:
-- title: Technology overview
-  items: 
-  - type: list
-    style: unordered
-    items:
-    - html: <a href="introduction">About Microsoft Academic Knowledge Exploration Service (MAKES)</a>
-- title: Get started
-  items:
-  - type: paragraph
-    text: Learn how to deploy, manage, and use MAKES.
-  - type: list
-    style: unordered
-    items:
-    - html: <a href="get-started-setup-provisioning">Sign up for Microsoft Academic Knowledge Exploration Service (MAKES)</a>
-    - html: <a href="get-started-create-api-instances">Deploy a basic MAKES instance using the provided scripts</a>
-- title: How to
-  items:
-  - type: paragraph
-    text: Learn how to upgrade and manage MAKES
-  - type: list
-    style: unordered
-    items:
-    - html: <a href="how-to-deploy-new-indexes-to-makes">How to deploy new releases to an existing instance of MAKES</a>
-    - html: <a href="how-to-delete-api-instances">How to delete a MAKES deployment</a>
-    - html: <a href="how-to-create-index-from-mag">Create custom MAKES index from a MAG release</a>
-    
-    
\ No newline at end of file
+  author: DarrinEide
+  ms.author: darrine
+  ms.date: 7/31/2020
+
+landingContent:
+  - title: About MAKES
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: What is MAKES?
+            url: introduction.md
+      - linkListType: concept
+        links:
+          - text: How MAKES understands natural language queries
+            url: concept-queries.md
+
+  - title: Get started
```

Comment:
> Get started [](start = 10, length = 12)

Get Started section already exist on the main navigation rail. How about moving the two tutorials under "Get started" section to "How-To" and have this page be the entry point of "Get-Started"?

Reply:
Hmm... I think having a "getting started" section is important in the navigation rail, as it's something folks are accustomed to looking for.

How about instead we simply rename this section on the landing page to "Top tasks"?

---
In reply to: [463834513](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/169#discussion_r463834513) [](ancestors = 463834513)